docs.json

```diff
@@ -56,6 +56,7 @@
             "group": "Browsers",
             "pages": [
               "browsers/create-a-browser",
+              "browsers/playwright-execution",
```

Comment:
@masnwilliams can you move this above or below computer controls in the navbar? Feels like that's the closest related thing?